src/locales/en/messages.json

```diff
@@ -3562,6 +3562,21 @@
   "personalOwnershipSubmitError": {
     "message": "Due to an Enterprise Policy, you are restricted from saving items to your personal vault. Change the Ownership option to an organization and choose from available Collections."
   },
+  "disableSend": {
+    "message": "Disable Send"
+  },
+  "disableSendPolicyDesc": {
+    "message": "Do not allow users to create or edit Bitwarden Sends. Deleting existing Sends is still allowed."
+  },
+  "disableSendExemption": {
+    "message": "Organization users that can manage the organization's policies are exempt from this policy's enforcement."
+  },
+  "sendDisabled": {
+    "message": "Send disabled"
+  },
+  "sendDisabledWarning": {
+    "message": "Due to an Enterprise Policy, you are restricted to only Deleting Sends."
```

Comment:
Do we capitalize "Enterprise Policy" like this in other policy warning messages, same with the action Deleting. If we can say "deleting a Send" that would be awesome too IMO

Reply:
I got the capitalization from personal ownership messaging, but it looks like single org doesn't capitalize. Let's decide which we like and I'll change all three

Reply:
Thanks for checking, let's go with non-capitalized everything except for "Send" which is a proper noun/brand